lib/ui/screens/collectible_found/collectible_found_screen.dart

```diff
@@ -143,24 +140,22 @@ class CollectibleFoundScreen extends StatelessWidget {
             child: child,
           ),
         )
-        .scale(begin: Offset(0.3, 0.3), duration: t * 2, curve: Curves.easeOutExpo, alignment: Alignment(0, 0.7));
+        .scale(begin: Offset(0.3, 0.3), duration: $styles.times.med, curve: Curves.easeOutExpo, alignment: Alignment(0, 0.7));
   }
```

Comment:
I would just leave this whole method as-is, it should just work as it's all tied into `styles.times`?

There seems to be a number of these, can you do a pass and make sure we're only changing code when we have to? If it works as-is, leave it as is. That will make it a bit easier for me to review what's left.